inference-engine/src/transformations/include/transformations/tensor_iterator_transformations/convert_tensor_iterator_to_sequence.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include <vector>
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+    class TRANSFORMATIONS_API ConvertTensorIteratorToLSTMSequence;
+    class TRANSFORMATIONS_API ConvertTensorIteratorToRNNSequence;
+    class TRANSFORMATIONS_API ConvertTensorIteratorToGRUSequence;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
```

Comment:
Add description for every transformation class

Reply:
these changes were moved to a separate branch